frontend/workflows/projectCatalog/src/details/components/card.tsx

```diff
@@ -50,16 +50,16 @@ interface CardProps extends CatalogDetailsCard, BaseCardProps {}
 const StyledCard = styled(Card)(({ theme }: { theme: Theme }) => ({
   width: "100%",
   height: "100%",
-  padding: theme.spacing(theme.clutch.spacing.base),
+  padding: theme.spacing("base"),
 }));
 
 const StyledGrid = styled(Grid)({
   height: "fit-content",
 });
 
 const StyledProgressContainer = styled("div")(({ theme }: { theme: Theme }) => ({
-  marginBottom: theme.spacing(theme.clutch.spacing.sm),
-  marginTop: theme.spacing(-theme.clutch.spacing.base),
+  marginBottom: theme.spacing("sm"),
+  marginTop: theme.spacing("-16px"),
```

Comment:
This could be done, right?
```suggestion
marginTop: theme.spacing(-2),
```